recipes/doxygen/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.9.7":
+    url: "https://www.doxygen.nl/files/doxygen-1.9.7.repack.src.tar.gz"
```

Comment:
The [original tarball](https://sourceforge.net/projects/doxygen/files/doxygen-1.9.7.src.tar.gz) has an issue, see https://github.com/doxygen/doxygen/issues/10113.

Thanks @dawikur for the hint!